src/govuk/components/character-count/character-count.yaml

```diff
@@ -141,3 +145,12 @@ examples:
       threshold: 75
       label:
         text: Full address
+
+  - name: with count message class
```

Comment:
Where possible we try to add tests rather than review application demonstrations for non-visible functionality. So we remove this example and test this with Jest rather than having to check the review application.